articles/container-apps/tutorial-code-to-cloud.md

```diff
@@ -183,8 +286,10 @@ az acr build --registry $ACR_NAME --image $API_NAME .
 
 # [Azure PowerShell](#tab/azure-powershell)
 
-```azurepowershell
-az acr build --registry $ACRName --image $APIName .
+The `az acr build` command does not have a PowerShell equivalent, but can be run in PowerShell.
+
+```powershell
+az acr build --registry $AcrName --image $APIName .
```

Comment:
If  you haven't used `acr` up until this point, doesn't the reader need to login first?

Reply:
True, I'll check and add that.